src/backend/trace.rs

```diff
@@ -336,6 +374,18 @@ impl TraceData {
                 let dur = probe.duration();
                 let dur_ms = dur.as_secs_f64() * 1000_f64;
                 hop.total_time += dur;
+                //Before last is set use it to set jitter
+                let last_ms = hop.last_ms().map_or(0f64, |d|d);
+                let jitter_ms = (last_ms - dur_ms).abs();
+                let jitter_dur = Duration::from_millis(jitter_ms as u64);
```

Comment:
I wonder if we can achieve this calculation without the round trip from `Duration` to `f64` via `u64`?

Reply:
let jitter_dur = Duration::from_secs_f64(jitter_ms / 1000_f64);
It's now dividing the millis by 1000 i.e. moving the decimal place.  I see this being done on other durations and should be safe.